commands/src/api/mod.rs

```diff
@@ -12,16 +12,35 @@ pub enum Error {
     DeserializeCommand,
 }
 
-/// API request packet type
+/// API message type
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(untagged)]
+pub enum Message {
+    /// Request
+    Request(Request),
+    /// Notification
+    Notification(Notification),
+}
+
+/// Request type
+/// Expect response from remote end
 #[derive(Debug, Serialize, Deserialize)]
 pub struct Request {
     /// Request body
     pub body: Body,
     /// Response correlation string
-    pub respond_to: Option<String>,
+    pub respond_to: String,
+}
+
+/// Notification
+/// Fire and forget packet
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Notification {
+    /// Request body
```

Comment:
Comment is copy-pasted from request :)
```suggestion
    /// Notification body
```